x-pack/legacy/plugins/maps/public/selectors/map_selectors.js

```diff
@@ -150,6 +150,8 @@ export const getLayerList = createSelector(
   }
 );
 
+export const getHiddenLayers = state => getLayerListRaw(state).filter(layer => !layer.visible);
```

Comment:
Can you use reselect for this selector definition since it needs to derive its state from `getLayerListRaw`?

```
export const getHiddenLayers = createSelector(
  getLayerListRaw,
  (layerDescriptorList) => {
    return layerDescriptorList.filter(layer => !layer.visible);
  }
);
```